app/assets/javascripts/arclight/context_navigation.js

```diff
@@ -101,9 +101,9 @@ class Placeholder {
 
 class ContextNavigation {
   constructor(el, originalParents = null, originalDocument = null) {
-    this.el = $(el);
-    this.data = this.el.data();
-    this.parentLi = this.el.parent();
+    this.el = el
+    this.data = $(this.el).data();
```

Comment:
can we not do `this.el.dataset` here?

Reply:
That's already done in #1210

Reply:
got it. I think if this gets rebased and conflicts are resolved it'll be a little easier to review

Reply:
It has been rebased.